donate/settings/development.py

```diff
@@ -41,6 +41,7 @@ def post_setup(cls):
 class ThunderbirdDevelopment(Development, ThunderbirdOverrides, Configuration):
     INSTALLED_APPS = ThunderbirdOverrides.INSTALLED_APPS + Development.INSTALLED_APPS
     FRONTEND = ThunderbirdOverrides.FRONTEND
+    CURRENCIES = ThunderbirdOverrides.CURRENCIES
```

Comment:
Is this line necessary? You have already applied that override in the parent class `ThunderbirdOverrides`. It seems like the child classes should inherit that.

Reply:
Hi Tibor! It appears that it is:
- When I take this line out, the minimum donation amount is then reverted back to $10
- Also, I can see that the other overrides on the parent class `ThunderbirdOverrides` also have to redefine this line in every file, for example `FRONTEND`.